lmfdb/templates/base.html

```diff
@@ -39,17 +39,18 @@
 
     <script type="text/javascript" src="https://www.google.com/jsapi"></script>
 
+
     <link rel="stylesheet"
-          href="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.css"
-          integrity="sha384-9eLZqc9ds8eNjO3TmqPeYcDj8n+Qfa4nuSiGYa6DjLNcv9BtN69ZIulL9+8CqC9Y"
+          href="https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.min.css"
+          integrity="sha384-dbVIfZGuN1Yq7/1Ocstc1lUEm+AT+/rCkibIcC/OmWo5f0EA48Vf8CytHzGrSwbQ"
           crossorigin="anonymous">
     <script defer
-          src="https://cdn.jsdelivr.net/npm/katex@0.10.0/dist/katex.min.js"
-          integrity="sha384-K3vbOmF2BtaVai+Qk37uypf7VrgBubhQreNQe9aGsz9lB63dIFiQVlJbr92dw2Lx"
+          src="https://cdn.jsdelivr.net/npm/katex@0.10.1/dist/katex.min.js"
+          integrity="2BKqo+exmr9su6dir+qCw08N2ZKRucY4PrGQPPWU1A7FtlCGjmEGFqXCv5nyM5Ij"
```

Comment:
There are no sha384- prefixes on these last two integrity hashes, is that intentional?

Reply:
Not intentional. I have just fixed it